File: src/codes/Users_Manual/modulation/fourier_2.cpp
//~#include<autd3.hpp>
//~int main() {
const auto m = autd3::modulation::Sine(100 * autd3::Hz) +
               autd3::modulation::Sine(150 * autd3::Hz)
                   .with_phase(autd3::pi / 2.0 * autd3::rad);
//~return 0; }

File: src/codes/Users_Manual/modulation/fourier_2.rs
# extern crate autd3;
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main()  {
let m = Sine::new(100 * Hz) + Sine::new(150 * Hz).with_phase(PI / 2.0 * rad);
# }